### PR TITLE
Index builder 0.0.15-alpha.2 release

### PR DESCRIPTION
**Pull request by dzhelezov**
- Fixed comments for PR-58, PR-55
- More bullet-proof indexer head updates
- Merged Indexer tests

### CODE REVIEW

File: substrate-query-framework/index-builder/src/model/QueryEvent.ts [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+  indexInBlock: number;
+  event_name: string;
+  event_method: string;
+  event_params: EventParameters;
+  extrinsic?: Extrinsic;
+  index: number
+}
 
-export class QueryEvent {
+export class QueryEventImpl implements QueryEvent {
```
Comment (metmirr) [inline substrate-query-framework/index-builder/src/model/QueryEvent.ts:20]: Would be better to rename them names as follows `IQueryEvent` and `QueryEvent` and converting property names to camelCase?

File: substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts [hunk 2]
```diff
@@ -55,6 +65,31 @@ export class SubstrateEventEntity {
   @JoinColumn()
   extrinsic?: SubstrateExtrinsicEntity;
 
+  // Warthog Fields
```
Comment (metmirr) [inline substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts:68]: We can move these fields to an abstract class `WarthogFields` and then extends our entities from this class.

File: substrate-query-framework/index-builder/src/index.ts [hunk 3]
```diff
@@ -9,8 +9,12 @@ export * from './node';
 export * from './substrate';
 export * from './db';
 export * from './processor';
+<<<<<<< Updated upstream
```
Comment (metmirr) [inline substrate-query-framework/index-builder/src/index.ts]: Probably miss this

File: substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts (hunk 2 above)
Comment (dzhelezov) [reply substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts]: Done

File: substrate-query-framework/index-builder/src/model/QueryEvent.ts (hunk 1 above)
Comment (dzhelezov) [reply substrate-query-framework/index-builder/src/model/QueryEvent.ts]: Done

Comment (metmirr) [reply substrate-query-framework/index-builder/src/model/QueryEvent.ts]: 👍

File: substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts (hunk 2 above)
Comment (metmirr) [reply substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts]: 👍

**Review by metmirr — APPROVED** [review]
LGTM!

**Pull request merged**